### DISCUSSION TITLE
Unable to get torque from Induction motor

### DISCUSSION THREAD

**Discussion by mssr86** [Eomys/pyleecan#649] [Q&A]
Hi,
I am trying to use the python script to simulate a SCIM. I have set slip and felec but there is no torque being generated whatever be the current I set. Can any one help me by looking at the script if there is any setting I am missing? The resultant rotor current is always 0A whatever slip i set. Not much help exisits on the forums or help for induction motor scripts that i could find.
[4W Pole IM.zip](https://github.com/Eomys/pyleecan/files/12679811/4W.Pole.IM.zip)

**Comment by SebGue** [comment]
Hi,

the short answer is, since you are running MagFEMM simulation you have to impose rotor currents to get non zero torque. 
MagFEMM is a static simulation so slip and frequency is disregarded.
Depending on your actual goals, you may use a combination of MagFEMM (for parameter estimation) and Electrical (with EEC_SCIM) to get some results. You could have a look at the tests "test_EEC_ELUT_SCIM.py" and "test_EEC_SCIM.py".

If you need further help, I will try to find some time.

Best regards, Sebastian
